mlir-tensorrt/executor/lib/Runtime/Backend/Lua/Modules/TensorRT/TensorRTModule.cpp

```diff
@@ -341,10 +399,13 @@ prepareBuffers(const AllocTracker &allocTracker,
 
 static Status enqueueV3Wrapper(AllocTracker &tracker,
```

Comment:
This function got a lot more complex with the addition of `outputDescriptors`. I think we should just have two separate functions -- keep the original (which wasn't very big anyway) and just have a separate function for the situation where TensorRT is going to allocate the outputs.